### PR TITLE
Kfdef 0.7.1

### PR DESCRIPTION
**Pull request by lluunn**
**Which issue is resolved by this Pull Request:**
https://github.com/kubeflow/kfserving/issues/568
https://github.com/kubeflow/kfctl/issues/124

**Description of your changes:**
Add 0.7.1 kfdef files

```
$ diff kfctl_gcp_iap.0.7.0.yaml kfctl_gcp_iap.0.7.1.yaml
399c399
<     uri: https://github.com/kubeflow/manifests/archive/c0e81bedec9a4df8acf568cc5ccacc4bc05a3b38.tar.gz
---
>     uri: https://github.com/kubeflow/manifests/archive/v0.7-branch.tar.gz
402c402
<   version: v0.7.0
---
>   version: v0.7.1
```

/cc @jlewi @richardsliu 

**Checklist:**
- [ ] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate`
    3. `make test`

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/656)

### CODE REVIEW

File: kfdef/kfctl_anthos.0.7.1.yaml [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+  - kustomizeConfig:
+      overlays:
+      - application
+      repoRef:
+        name: manifests
+        path: seldon/seldon-core-operator
+    name: seldon-core-operator
+  repos:
+  - name: manifests
+    uri: https://github.com/kubeflow/manifests/archive/918b91d1efcd4bbdb718e5f11a2ee8e13b2f7c14.tar.gz
```
Comment (richardsliu) [inline kfdef/kfctl_anthos.0.7.1.yaml]: Should we use a commit hash here? What if we need to create more fixes for v0.7.1?

Comment (lluunn) [reply kfdef/kfctl_anthos.0.7.1.yaml]: This is a commit hash right?

Comment (lluunn) [reply kfdef/kfctl_anthos.0.7.1.yaml]: 918b91d1efcd4bbdb718e5f11a2ee8e13b2f7c14

Comment (richardsliu) [reply kfdef/kfctl_anthos.0.7.1.yaml]: I meant should we be using commit hash or branch head here? If we make another cherrypick then we would need to update this right?

Comment (lluunn) [reply kfdef/kfctl_anthos.0.7.1.yaml]: done

**Pull request merged**